machine/drivers/virtualbox.md

```diff
@@ -37,6 +37,7 @@ The size of the VM's disk can be configured this way:
 -   `--virtualbox-no-share`: Disable the mount of your home directory
 -   `--virtualbox-no-dns-proxy`: Disable proxying all DNS requests to the host (Boolean value, default to false)
 -   `--virtualbox-no-vtx-check`: Disable checking for the availability of hardware virtualization before the vm is started
+-   `--virtualbox-share-folder`: Override the default shared folder in the form "path:name"
```

Comment:
From: https://github.com/docker/machine/blob/2b352ca045eab9e28bc045841fd148b752eb18df/drivers/virtualbox/virtualbox.go#L197

How about: 

`Mount the specified directory instead of the default home location. Format: dir:name`